mlbstatsapi/mlbdataadapter.py

```diff
@@ -7,7 +7,10 @@ class MlbResult:
     def __init__(self, status_code: int, message: str, data: Dict = {}):
         self.status_code = int(status_code)
         self.message = str(message)
-        self.data = data if data else {}
+        if 'copyright' in data:
+            self.data = data if data.pop('copyright') else {}  # this can be refactored
+        else:
+            self.data = {}
```

Comment:
Are you checking for if the return has 'copyright' to one, remove it and two if not in the return then something has gone, self.data is now empty?

Reply:
I couldn't figure out a good way to write this. I just want to remove the copyright key from all mlbresults. It's useless for us.

Reply:
Any good ideas?

Reply:
As it stands, this would not pass tests since a few of our base classes expect copywrite and a field. Did this past test for you with this change?

Reply:
Fixed them. The CI/CD github workflow pytest run passes.

Reply:
Perfect!

Reply:
@KCNilssen you didn't resolve!